src/components/MovieDetails/MovieDetails.js

```diff
@@ -66,51 +65,43 @@ class MovieDetails extends Component {
         })
       }
 
+    formatRuntime = (runtime) => {
+     return !runtime ? 'Not Reported' : runtime + " mins"
+    }
+
+
     render() {
         
-        // const { title, release_date, backdrop_path, overview, genres, budget, revenue, runtime, average_rating } = this.state.movieInfo
-        // let formattedRating
-        // let formattedGenre
-        // let formattedBudget
-        // let formattedRevenue
+        const { id, title, release_date, backdrop_path, poster_path, overview, genres, budget, revenue, tagline, average_rating, runtime} = this.state.movieInfo
```

Comment:
This destructuring looks so much better than what we had, with the busy "this.state.movieInfo"